a3/terraform/modules/cluster/gke-beta/scripts/gke_cluster.sh

```diff
@@ -29,31 +29,31 @@ gke_cluster::create () {
         return 1
     } >&2
 
+    echo "Updating default subnet" >&2
+    gcloud compute networks subnets update default \
+      --region "${region}" \
+      --add-secondary-ranges="${cluster_name}-pods=10.150.0.0/21,${cluster_name}-services=10.150.8.0/21"
+
     echo "Creating cluster '${cluster_name}'..." >&2
     gcloud beta container clusters create "${cluster_name}" \
-        --cluster-version="${version}" \
-        --enable-ip-alias \
+        --image-type=CUSTOM_CONTAINERD \
+        --no-enable-autoupgrade \
+        --no-enable-shielded-nodes \
         --enable-dataplane-v2 \
+        --region="${region}" \
+        --enable-ip-alias \
         --enable-multi-networking \
-        --num-nodes='1' \
-        --network="${network_name}" \
+        --num-nodes='15' \
```

Comment:
just curious where is this "15" comes from? is this a random number?

Reply:
So far this is random and I am copying this from our existing script. This is the number of e2-medium nodes in the default pool. and we need the default-pool for hosting DNS. We dont have any consensus on what this node count should be. We would need more nodes if we create a huge GKE cluster but I feel 15 is a good starting point. Please advice if you feel differently.

Reply:
sounds good. each kube-dns would request 0.15 CPU so we wouldnt need to many default nodes there.